### DISCUSSION TITLE
Error when viewing inline PR comments - bad argument #6 to 'format' (number expected, got userdata)

### DISCUSSION THREAD

**Discussion by DexterShepherd** [pwntester/octo.nvim#312] [Q&A] [answered]
Hey all,

I see this error when moving the cursor over an inline comment in a PR. 

```
Error detected while processing CursorMoved Autocommands for "*":
Error executing lua callback: ...nvim/plugged/octo.nvim/lua/octo/reviews/thread-panel.lua:99: bad argument #6 to 'format' (number expected, got userdata)
stack traceback:
        [C]: in function 'format'
        ...nvim/plugged/octo.nvim/lua/octo/reviews/thread-panel.lua:99: in function 'create_thread_buffer'
        ...nvim/plugged/octo.nvim/lua/octo/reviews/thread-panel.lua:64: in function 'show_review_threads'
        ...local/share/nvim/plugged/octo.nvim/lua/octo/autocmds.lua:42: in function <...local/share/nvim/plugged/octo.nvim/lua/octo/autocmds.lua:41>v
```

Happy to make an issue if this is an actual bug, I'm a little new to Lua plugins so it's also possible it's something on my end.

macos 12.4
nvim 0.7.2

<img width="1422" alt="Screen Shot 2022-07-30 at 1 03 05 PM" src="https://user-images.githubusercontent.com/7800175/181994765-577cf6a5-4ea5-4772-b5ee-7aaf86d717db.png">

**Comment by pwntester** [comment, marked as answer]
Sorry, I missed this question. Can you please make an issue out of it and add details to reproduce it (if possible)?
